src/OIDCCryptUtils/JSONWebKey.purs

```diff
@@ -0,0 +1,11 @@
+module OIDCCryptUtils.JSONWebKey (JSONWebKey()) where
+
+import Prelude (pure, (<<<))
+import Data.Argonaut as JS
+
+foreign import
+  data JSONWebKey :: *
```

Comment:
thats a good point. would it have the same underlying representation? i.e. would it be a simple javascript object?